igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/BuildController.groovy

```diff
@@ -184,10 +184,7 @@ class BuildController {
 
             queuedLocation.split('/')[-1]
         } else if (buildMasters.filteredMap(BuildServiceProvider.TRAVIS).containsKey(master)) {
-            if (requestParams) {
-                return buildMasters.map[master].triggerBuildWithParameters(job, requestParams)
-            }
-            return buildMasters.map[master].triggerBuild(job)
+            return buildMasters.map[master].triggerBuildWithParameters(job, requestParams)
```

Comment:
Since it doesn't look like `triggerBuildWithParameters` was changed, does it gracefully handle `null` & empty `requestParams`?